core/src/test/scala/com/yahoo/maha/core/query/oracle/OracleQueryGeneratorTest.scala

```diff
@@ -1896,7 +1896,7 @@ class OracleQueryGeneratorTest extends BaseOracleQueryGeneratorTest {
         |                  FROM
         |               ( (SELECT  advertiser_id, campaign_id, ad_group_id, id
         |            FROM ad_dim_oracle
-        |            WHERE (id = ad_group_id) AND (advertiser_id = 12345)
+        |            WHERE (advertiser_id = 12345) AND (id = ad_group_id)
```

Comment:
can you check why the order has changed in the where clause ? 
It will too much work to change our real tests

Reply:
Because of change in filter ordering here-
https://github.com/ritvikjaiswal/maha/blob/mult_filt_same_col/core/src/main/scala/com/yahoo/maha/core/FilterOperation.scala#L1688